Menu (#2957)
jpmorganchase/salt-ds: site/docs/components/menu/index.mdx

---
title: Menu
data:

>   description: "The `Menu` component is a container that presents a selection of choices or actions for a user to choose from. It can be opened by a customizable trigger, e.g., an interactive element such as a button, or an interaction, such as right mouse click or keyboard shortcut."

Reviewer: ```suggestion
  description: "`Menu` is a container that presents a selection of choices or actions for a user to choose from. The user opens it with a customizable trigger, e.g., an interactive element such as a button, or an interaction, such as right mouse click or keyboard shortcut."
```
